fastlane/lib/fastlane/actions/changelog_from_git_commits.rb

```diff
@@ -79,7 +79,7 @@ def self.available_options
                                        env_name: 'FL_CHANGELOG_FROM_GIT_COMMITS_BETWEEN',
                                        description: 'Array containing two Git revision values between which to collect messages, you mustn\'t use it with :commits_count key at the same time',
                                        optional: true,
-                                       is_string: false,
+                                       skip_type_validation: true, # allow Array, String both
```

Comment:
I think making `type: Array` actually allows both a String and an Array of Strings so I think this is what we want here 😊 

```suggestion
                                       type: Array, # allow Array, String both
```